crates/hc_sandbox/tests/cli.rs

```diff
@@ -269,9 +269,10 @@ async fn generate_sandbox_memproof_deferred_and_call_list_dna() {
     assert!(exit_code.success());
 }
 
-/// Create a new default sandbox which should have DPKI enabled in the conductor config.
+/// Create a new default sandbox which should have DPKI disabled in the conductor config.
```

Comment:
For the time being the default is that hc sandboxes have DPKI disabled.

Reply:
Please can this test be duplicated, one with the feature on and one with the feature off to test the defaults in each case? The default when the feature is enabled shouldn't have changed right?